test/models/trail_test.rb

```diff
@@ -1,5 +1,48 @@
 require "test_helper"
 
 class TrailTest < ActiveSupport::TestCase
+  def setup
+    trail1 = Trail.create(name: "howdy hike",
+                          city: "Denver",
+                         state: "CO",
```

Comment:
Personally, I like the way you aligned the Trail.create hash, the hound, not so much.